tests/puppeteer/pages/BO/BO_commonPage.js

```diff
@@ -0,0 +1,75 @@
+const CommonPage = require('../commonPage');
+
+module.exports = class BO_COMMONPAGE extends CommonPage {
+  constructor(page) {
+    super(page);
+
+    this.pageTitle = 'Dashboard •';
+
+    // top navbar
+    this.headerLogoImage = '#header_logo';
+    this.userProfileIcon = '#employee_infos';
+    this.userProfileLogoutLink = 'a#header_logout';
+
+    // left navbar
+    // SELL
+    this.ordersParentLink = 'li#subtab-AdminParentOrders';
+    this.ordersLink = '#subtab-AdminOrders>a';
+
+    this.productsParentLink = 'li#subtab-AdminCatalog';
+    this.productsLink = '#subtab-AdminProducts>a';
+
+    this.customersParentLink = 'li#subtab-AdminParentCustomer';
+    this.customersLink = '#subtab-AdminCustomers>a';
+
+    // welcome module
+    this.onboardingCloseButton = 'button.onboarding-button-shut-down';
+    this.onboardingStopButton = 'a.onboarding-button-stop';
+  }
+
+  /*
+  Methods
+   */
+  /**
+   * Open a subMenu if closed and click on a sublink
+   * @param blockSelector
+   * @param parentSelector
+   * @param linkSelector
+   * @returns {Promise<void>}
+   */
+  async goToSubMenu(parentSelector, linkSelector) {
+    if (await this.elementVisible(linkSelector)) {
+      await this.page.click(linkSelector);
+    } else {
+      // open the block
+      await this.page.click(parentSelector);
+      await this.page.waitForSelector(`${parentSelector}.open`, {visible: true});
+      await this.page.click(linkSelector);
+    }
+    this.page.waitFor(500);
```

Comment:
Maybe doing this instead
```js
//Selectors
    this.ordersLink = '#subtab-AdminOrders';
    this.productsLink = '#subtab-AdminProducts';
    this.customersLink = '#subtab-AdminCustomers';

// In goToSubMenu function
    await this.page.waitForSelector(`${linkSelector}.-active`, {visible: true});
```

Reply:
Doesn't work sadly. I'll try something else.

Reply:
It works if I change the Link selector to remove the `>a` part